doc/command_ref.rst

```diff
@@ -243,16 +243,16 @@ Check Update Command
 -------------
 Clean Command
 -------------
-Performs cleanup of temporary files for the currently enabled repositories.
+Performs cleanup of temporary files kept for repositories. This includes any
+such data left behind from disabled or removed repositories as well.
```

Comment:
This includes any such data left behind from disabled or removed repositories regardless what release version is set.